vm/evaluation_test.go

```diff
@@ -1509,6 +1509,44 @@ func TestMethodCallWithBlockArgument(t *testing.T) {
 			  ten + 20
 			end
 			`, 30},
+		// Get Block
+		{`
```

Comment:
@saveriomiroddi Are these what you need to implement #537?

Reply:
> Are these what you need to implement #537?

@st0012 yes, I think so!